rest-service/manager_rest/storage/resource_models.py

```diff
@@ -657,6 +665,19 @@ def node_instance(cls):
     node_id = association_proxy('node_instance', 'node_id')
     deployment_id = association_proxy('node_instance', 'deployment_id')
 
+    @property
+    def host_id(self):
+        return self.name
+
+    node = node_id
+    deployment = deployment_id
+    node_ids = node_id
+    node_instance_ids = node_instance_id
```

Comment:
1. Make sure it doesn't affect the migration file.
2. Maybe the fields relevant to the response can be initialise  in the `to_response` function?